project/Build.scala

```diff
@@ -190,19 +191,6 @@ object Scrooge extends Build {
       finagle("core") exclude("org.mockito", "mockito-all"),
       finagle("thrift") % "test"
     ),
-    libraryDependencies <++= scalaVersion({
```

Comment:
There are different versions of things at play here:

- Java 7 or 8
- Scala 2.10 or 2.11
- Mustache.java 0.8.17 or 0.9

We need to support Scala 2.11 with Java 7. Mustache.java from 0.9 onwards only supports Java 8, so we need version 0.8.17 of Mustache.java. Scala-extensions has been published for Scala 2.10, but for Scala 2.11 the only scala-extensions published are for Mustache.java 0.9 (and therefore Java 8). That is the reason why ScalaObjectHandler is inlined - we need ScalaObjectHandler for Java 7 and Scala 2.11.

The problem with the Scrooge build was that it pattern-matched `scalaVersion` and set library versions accordingly, but when building for Scala 2.11 it would only work with Java 8.
Once Scrooge is declared Java-8-only, it can use Mustache.java 0.9 and Scala-extensions 2.11.